lib/patientFinder/utils.rb

```diff
@@ -0,0 +1,165 @@
+require 'rubygems'
+require 'nokogiri'
+require 'health-data-standards'
+require 'zip/zipfilesystem'
+require 'fileutils'
+require 'health-data-standards'
+require 'pry'
+Mongoid.load!("mongoid.yml", :development)
+
+module PatientFinder
+# a collection of utilities grouped together under the same roof
+    class Utils
+        
+        #some of records are created ambiguous on purpose (fr example some of them contain only encounters) - in order to disambiguate human intervention is needed - the following hash resolves the ambiguos cases
+        AMBIGUOUS_PATIENTS = {["ee85c89f24946e2ddca12c6edc5181dc", "eb6af018e11320f5c163624beb83767d"] => "ee85c89f24946e2ddca12c6edc5181dc"}            
```

Comment:
Just a formatting comment: usually hashes are created all within one set of curly braces, rather than doing one value per line like you have